app/decorators/category_decorator.rb

```diff
@@ -20,10 +20,19 @@ def canonical_url(posts)
   end
 
   def page_title(posts)
+    if site.category_title_format.present?
+      begin
+        title = site.category_title_format % {category_name: name}
+      rescue KeyError
```

Comment:
I added code that restrict `Site#category_title_format` in 747edba.
This is the safe net if my implementation of `Site#validate_category_title_format` have bugs.
This code improves robustness.